README.md

```diff
@@ -4,30 +4,312 @@
 [![Docker Repository on Quay](https://quay.io/repository/redhat-aqe/sync2jira/status "Docker Repository on Quay")](https://quay.io/repository/redhat-aqe/sync2jira)
 [![Build Status](https://travis-ci.org/release-engineering/Sync2Jira.svg?branch=master)](https://travis-ci.org/release-engineering/Sync2Jira)
 [![Coverage Status](https://coveralls.io/repos/github/release-engineering/Sync2Jira/badge.svg?branch=master)](https://coveralls.io/github/release-engineering/Sync2Jira?branch=master)
-![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)
+![Python 3.9+](https://img.shields.io/badge/python-3.9+-blue.svg)
 
-## What is Sync2Jira?
+## Overview
 
-This is a process that listens to activity on upstream repos on
-GitHub via fedmsg, and syncs new issues there to a Jira instance elsewhere.
+Sync2Jira is a service that listens to activity on upstream GitHub repositories via fedmsg and automatically syncs issues and pull requests to downstream JIRA instances. It provides real-time synchronization capabilities, ensuring that your JIRA project stays up-to-date with upstream development activity.
 
-## Documentation
+### Key Features
+
+- **Real-time Synchronization**: Listens to fedmsg events from GitHub for immediate updates
+- **Bidirectional Sync**: Supports both issues and pull requests
```

Comment:
"Bidirectional" is the wrong word:  that will make people think that they can update the Jira issue and see the effect in the GitHub issue...which is something (some) people want and we're not likely to deliver (because it's tricky).

I can't come up with a better word ("multimodal" doesn't quite work), so maybe just remove the summary prefix from the line?